Packs/CortexDataLake/Playbooks/Cortex_Data_Lake_-_File_Indicators_Hunting.yml

```diff
@@ -0,0 +1,1812 @@
+                iscontext: true
+          - - operator: notInList
+              left:
+                value:
+                  simple: CDL.Logging.Threat.SessionID
+                iscontext: true
+              right:
+                value:
+                  simple: CDL.HuntingResults.SessionID
+                iscontext: true
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -3210,
+          "y": 770
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "315":
+    id: "315"
+    taskid: 44b7e440-80e9-4a8c-87cf-6ce9aa866cf6
+    type: title
+    task:
+      id: 44b7e440-80e9-4a8c-87cf-6ce9aa866cf6
+      version: -1
+      name: Hash Indicators (SHA1, SHA256, and MD5)
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "316"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -2780,
+          "y": -150
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "316":
+    id: "316"
+    taskid: be812b5e-d289-426d-8b57-5d8867687cbf
+    type: condition
+    task:
+      id: be812b5e-d289-426d-8b57-5d8867687cbf
+      version: -1
+      name: Any hash to hunt for?
+      description: Checks whether any MD5, SHA256, or SHA1 hashes are available for threat hunting.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "311"
+      "yes":
+      - "333"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.MD5
+            iscontext: true
+          right:
+            value: {}
+        - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.SHA256
+            iscontext: true
+        - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.SHA1
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -2780,
+          "y": -20
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "333":
+    id: "333"
+    taskid: ff5e0aab-37dc-42b0-8d2e-2370443306ab
+    type: playbook
+    task:
+      id: ff5e0aab-37dc-42b0-8d2e-2370443306ab
+      version: -1
+      name: Convert file hash to corresponding hashes
```

Comment:
what is the playbook (missing from the PR) and also Im not sure its neded

Reply:
The 'Convert file hash to corresponding hashes' playbook is part of the 'Common Playbooks' pack and is used to convert MD5 and SHA1 hashes to SHA256 hashes.

As 'cdl-query-threat-logs' and 'cdl-query-file-data' accept only SHA256 hashes, I decided to add 'Convert file hash to corresponding hashes' as a sub-playbook, so MD5 and SHA1 inputs will also be supported.

Reply:
@altmannyarden 
Let me know if you want me to remove this sub-playbook. If so, I will have to remove the MD5 and SHA1 inputs as well.

Reply:
done